### PR TITLE
Prepare for itt reform

### PR DESCRIPTION
**Pull request by ralph-hawkins**
How the service will change to adapt to the new ITT policy. Including changes to routes and organisations.

### CODE REVIEW

File: app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  description:  How the service will change to adapt to the new ITT policy
+  date:         2024-06-19
+  tags:
+    - ITT reform
+    - routes
+    - providers
+    - hesa
+---
+
+To prepare Register trainee teachers (Register) for the changes brought by ITT reforms we needed to make updates to training routes and organisations. These changes will be in place for the beginning of the 2024 to 2025 academic year.
```
Comment (simonwhatley) [inline app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md]:
```suggestion
To prepare Register trainee teachers (Register) for the changes brought by initial teacher training (ITT) reforms, we needed to update training routes and organisations. These changes will be in place for the beginning of the 2024 to 2025 academic year.
```

File: app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Thia introduces 2 types of organisation that Register did not previously need to account for:
+
+1. Providers who lost their accreditation and stopped all training
+2. Providers who lost their accreditation and have become lead partners
+
+Both these new types of organisation will need to be:
+
+- prevented from creating new trainees
+- able to view and edit trainees who started before 2024/25 (edit access is removed from all trainees once teacher status has been awarded)
```
Comment (simonwhatley) [inline app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md]:
```suggestion
- able to view and edit trainees who started before the 2024 to 2025 academic year (edit access is removed from all trainees once teacher status has been awarded)
```

File: app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+| Accredited provider | Accredited provider | Read&#8209;write | Read&#8209;write |
+| Accredited provider | Lead partner        | Read&#8209;write | Read&#8209;only  |
+| Accredited provider | Stoped training     | Read&#8209;write | None             |
+| Lead school         | Lead partner        | Read&#8209;only  | Read&#8209;only  |
+
+### Adding and updating trainee records through the Higher Education Statistics Agency (HESA)
+
+Because the HESA data does not allow for the notion of ‘accredited&nbsp;provider’ or ‘lead&nbsp;partner’, lead&nbsp;partners will be permitted to use their student record system and HESA to submit records.
+
+When a lead&nbsp;partner uses HESA to submit data, the Register app will automatically attribute records to the accredited&nbsp;provider the lead&nbsp;partner is working with.
```
Comment (simonwhatley) [inline app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md]:
```suggestion
When a lead partner uses HESA to submit data, Register will automatically attribute records to the accredited provider the lead partner is working with.
```

**Review by simonwhatley — CHANGES_REQUESTED** [review]

File: app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md (hunk 3 above)
Comment (ralph-hawkins) [reply app/posts/register-trainee-teachers/2024-06-25-preparing-for-itt-reform.md]: I'll change ‘the Register app’ to ’Register‘, but keep the non-breaking spaces

**Review by simonwhatley — APPROVED** [review]

**Pull request merged**